agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/protocol_generated/ModelProvider.kt

```diff
@@ -6,6 +6,7 @@ data class ModelProvider(
   val codyProOnly: Boolean? = null,
   val provider: String? = null,
   val title: String? = null,
+  val contextWindow: Int? = null,
```

Comment:
Put the (approximate) units in this name. You intend this as a character count or byte count, not a token count, but if people mix them up the type system won't save you!

Let's name it so reviewers can at least spot problems.

```suggestion
  val contextWindowSizeChars: Int? = null,
```